composer.json

```diff
@@ -23,33 +23,35 @@
         }
     ],
     "require": {
-        "php": "^5.3.3|^7.0",
-        "doctrine/common": "^2.3|^3.0",
-        "symfony/config": "^2.3|^3.0",
-        "symfony/http-kernel": "^2.3|^3.0",
-        "symfony/http-foundation": "^2.4|^3.0",
-        "symfony/dependency-injection": "^2.3|^3.0"
+        "php": "^5.4|^7.0",
+        "doctrine/common": "^2.5",
+        "symfony/config": "^2.8|^3.0",
```

Comment:
Since Symfony uses semver, it has no sense. If we want to support both ^2.* and ^3.*, we have to upgrade minimum version, otherwise is too complicated with deprecation points and new specifications.